lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -203,6 +207,12 @@ export class RedirectClient extends StandardInteractionClient {
                 this.logger.info(
                     "handleRedirectPromise called but there is no interaction in progress, returning null."
                 );
+                if (performanceClient && correlationId) {
+                    performanceClient?.addFields(
+                        { errorCode: "no_interaction_in_progress" },
```

Comment:
This will be emitted on every page load, perhaps multiple times and is 90% of the time expected. Is this desirable? Do we have a way to distinguish expected vs. unexpected?

Reply:
Thanks for the context, appreciate it. I am not sure I can come up with a way to distinguish expected vs unexpected. I don't think we can use `hash` as part of the unexpected condition. Happy to remove this one for now.

Reply:
You could check if hash (which would be hash passed in OR window.location.hash/search) contains known properties (there's a helper function that does this)

That would give you a pretty good idea, however, downside is that you're now parsing the url hash on every page load which could be expensive. Since this is adding some level of perf tracking, maybe we try it and if it proves to be too expensive reconsider.

Reply:
Checking hash for known properties could be an option, but i would rather avoid doing that. To me the perf regression outweighs the potential tracking benefits.